### DISCUSSION TITLE
Tooltip for p-slider

### DISCUSSION THREAD

**Discussion by KevinBurton** [primefaces/community#1724] [PrimeNG] [answered]
I would like to leverage two PrimeNg components, p-tooltip and p-slider. I would like the text of the tooltip to be the current value of the slider and I would like the tooltip to be attached to the handle of the slider. In addition I would like to have a range slider so there are two handles and two tooltips each following the respective handle and displaying the current value of the handle. I understand that this is not possible with the current library and some work will need to be done to extend Tooltip. It seems the information about extending Tooltip is out of date. My question is to either suggest a solution to my original question or point me to a way to extend Tooltip using Angular 17+.
Thank you.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for the feedback, it's a good combination. We'll wait for the upvotes to implement this as a feature.
